server/src/main/java/org/apache/iotdb/db/qp/executor/PlanExecutor.java

```diff
@@ -1097,6 +1098,13 @@ private void loadFile(File file, OperateFilePlan plan) throws QueryProcessExcept
       List<ChunkGroupMetadata> chunkGroupMetadataList = new ArrayList<>();
       try (TsFileSequenceReader reader = new TsFileSequenceReader(file.getAbsolutePath(), false)) {
         reader.selfCheck(schemaMap, chunkGroupMetadataList, false);
+        if (plan.getMetadataCheck() && !loadNewTsFileCheckMetadata(reader)) {
+          throw new QueryProcessException(
+              "loading tsfile metadata is inconsistent with existing timeseries.");
```

Comment:
print which timeseries is not consistent:

root.sg.d.s is INT32 in the loading TsFile but is FLOAT in IoTDB.

Reply:
i have moved this info to method "loadNewTsFileVerifyMetadata"